### DISCUSSION TITLE
On sign up pages, some text in the page seemed to blend with its background color

### DISCUSSION THREAD

**Discussion by typed-sigterm** [community/community#116544] [Copilot Conversations] [answered]
### Select Topic Area

General

### Body

Hi, I just got GitHub Student Developer Pack (thank you GitHub) and when signing up for GitHub Copilot, I noticed that some text in the page seemed to blend with its background color:

![image](https://github.com/community/community/assets/145281501/b49aa6e0-e0ac-4290-a487-9bc973b81577)

Both https://github.com/github-copilot/free_signup & https://github.com/github-copilot/signup/settings have this issue.

**Comment by vthanhduong** [comment, marked as answer]
i guess that maybe some attributes of the tags in the page's html code are assigned the wrong value or just some css mistakes.
in dark mode, this page still displays in light mode (as you mentioned).
so i took a look at the page's html code, and changed the "data-color-mode" attribute of a div tag containing it from "light" to "dark" and the page displayed correctly (in dark mode of course).
maybe GitHub will fix it soon.
![image](https://github.com/community/community/assets/92918544/b3860b40-8770-4910-8711-22b398231a4a)
